com.unity.netcode.gameobjects/Tests/Runtime/NetworkAnimator/NetworkAnimatorTests.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections;
+using NUnit.Framework;
+using Unity.Netcode.Components;
+using Unity.Netcode.RuntimeTests;
+using UnityEditor.Animations;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+
+namespace Unity.Netcode.RuntimeTest
+{
+    public class AnimatorTests : BaseMultiInstanceTest
+    {
+        protected override int NbClients => 1;
+        private GameObject m_TestPrefab;
+
+        [UnitySetUp]
+        public override IEnumerator Setup()
+        {
+            yield return StartSomeClientsAndServerWithPlayers(useHost: true, nbClients: NbClients,
+                updatePlayerPrefab: playerPrefab =>
+                {
+                    AnimatorController controller = Resources.Load("TestAnimatorController") as AnimatorController;
+                    var animator = playerPrefab.AddComponent<Animator>();
+                    animator.runtimeAnimatorController = controller;
+
+                    var theAnimator = playerPrefab.AddComponent<NetworkAnimator>();
+                    theAnimator.Animator = animator;
+                });
+        }
+
+        [UnityTest]
+        public IEnumerator AnimationStateSyncTest()
+        {
+            // This is the *SERVER VERSION* of the *CLIENT PLAYER*
+            var serverClientPlayerResult = new MultiInstanceHelpers.CoroutineResultWrapper<NetworkObject>();
+            yield return MultiInstanceHelpers.Run(MultiInstanceHelpers.GetNetworkObjectByRepresentation((x => x.IsPlayerObject && x.OwnerClientId == m_ClientNetworkManagers[0].LocalClientId), m_ServerNetworkManager, serverClientPlayerResult));
+            var serverPlayer = serverClientPlayerResult.Result.gameObject;
+            var serverAnimator =  serverPlayer.GetComponent<Animator>();
+
+            // This is the *CLIENT VERSION* of the *CLIENT PLAYER*
+            var clientClientPlayerResult = new MultiInstanceHelpers.CoroutineResultWrapper<NetworkObject>();
+            yield return MultiInstanceHelpers.Run(MultiInstanceHelpers.GetNetworkObjectByRepresentation((x => x.IsPlayerObject && x.OwnerClientId == m_ClientNetworkManagers[0].LocalClientId), m_ClientNetworkManagers[0], clientClientPlayerResult));
+            var clientPlayer = clientClientPlayerResult.Result.gameObject;
+            var clientAnimator = clientPlayer.GetComponent<Animator>();
+
+            var serverAnimatorStateInfo = serverAnimator.GetCurrentAnimatorStateInfo(0);
+            var clientAnimatorStateInfo = clientAnimator.GetCurrentAnimatorStateInfo(0);
+
+            // we start in the default state
+            Assert.True(serverAnimatorStateInfo.IsName("DefaultState"));
+            Assert.True(clientAnimatorStateInfo.IsName("DefaultState"));
+
+            serverAnimator.SetBool("AlphaParameter", true);
+
+            // (ugh)
+            yield return new WaitForSeconds(4.0f);
```

Comment:
@NoelStephensUnity as I was writing this I wondered if you would chime in on your recent testing changes